android/src/main/java/com/alpha0010/FileAccessModule.kt

```diff
@@ -25,7 +26,8 @@ class FileAccessModule(reactContext: ReactApplicationContext) : ReactContextBase
       "CacheDir" to reactApplicationContext.cacheDir.absolutePath,
       "DatabaseDir" to reactApplicationContext.getDatabasePath("FileAccessProbe").parent,
       "DocumentDir" to reactApplicationContext.filesDir.absolutePath,
-      "MainBundleDir" to reactApplicationContext.applicationInfo.dataDir
+      "MainBundleDir" to reactApplicationContext.applicationInfo.dataDir,
+      "DownloadDir", to Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS).getAbsolutePath()
```

Comment:
Documentation says [`getExternalStoragePublicDirectory()`](https://developer.android.com/reference/android/os/Environment#getExternalStoragePublicDirectory(java.lang.String)) is deprecated, and recommends [`getExternalFilesDir()`](https://developer.android.com/reference/android/content/Context#getExternalFilesDir(java.lang.String)).  The latter however, restricts file visibility to only the current app.  Does that still meet your needs for access to the downloads directory?